provider/testdata/data/json/displaynames/languages@1/sr-Latn.json

```diff
@@ -555,5 +558,19 @@
     "zun": "zuni",
     "zxx": "bez lingvističkog sadržaja",
     "zza": "zaza"
+  },
+  "short_names": {
```

Comment:
Thought: We could instead do this by putting the other lengths in their own data key, a supplement on top of the main data key. But, these are small (5-10%) of the data size, so probably not compelling.

Reply:
Acknowledged. Because these names are small percentage of the total data size, I am keeping this as is.